### DISCUSSION TITLE
How would you add a tag directive to an enum?

### DISCUSSION THREAD

**Discussion by jwpark511** [MichalLytek/type-graphql#1510] [General]
I want to emit my schema like this `enum SomeEnum @tag(name: "internal")`

For object and input types I can add `@Directive('@tag(name: "internal")')` and use instructions from https://typegraphql.com/docs/emit-schema.html to print the schema, but for enums I can't use the directive decorator since enums are not a class.

Any ideas/suggestions on how to do this?

**Comment by MichalLytek** [comment]
It's not supported yet. Directives are a hack in schema builder, so not implemented for all cases.
